styleguide/index.erb

```diff
@@ -632,6 +587,51 @@
       </li>
     </ul>
     <% end %>
+
+    <% styleguide_block  'Gallery - Mosaic' do %>
+    <ul class="gallery -quartet -mosaic">
+        <li>
```

Comment:
Extra indentation is because it's inside a `#gallery` wrapping div a little ways up. :)